docs/api.rst

```diff
@@ -231,7 +231,64 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+Token Swaps
+------------
+
+You can perform a token swap by using the token_swap endpoint. A swap consists of two users agreeing on atomically exchanging two different tokens at a particular exchange rate.
+
+By making a ``PUT`` request to ``/api/<version>/token_swaps/<target_address>/<identifier>`` you can either initiate or participate in a token swap with a specific user. The details, along with the role, come as part of the json payload.
+
+Example Request
```

Comment:
We are not giving curl examples anywhere else in the api documentation. Perhaps later we can do a pass on the document and add curl examples. It's a good idea, just don't think it should be part of this PR.